list/PageableList.js

```diff
@@ -265,41 +170,42 @@ define([
 		 */
 		_lastLoaded: -1,
 
+		/**
+		 * Visibility flag for previousPageLoader widget.
+		 * @member {boolean}
+		 * @default false
+		 * @private
+		 */
+		_showPreviousPageLoader: false,
+
+		 /**
+		 * Visibility flag for nextPageLoader widget.
+		 * @member {boolean}
+		 * @default false
+		 * @private
+		 */
+		_showNextPageLoader: false,
```

Comment:
There's an inconsistency here with the definition ( "Visibility flag for previousPageLoader widget.") vs. what these flag are actually doing, specifically that sometimes the previous page (and next page) buttons won't be shown even when `_showPreviousPageLoader` and `_showNextPageLoader` are both true.

It's probably better that these flags actually work as they say, so the template can just say:

```
<d-list-loader d-shown={{_showPreviousPageLoader}} ...
```

An alternate approach would be to keep the logic as-is but rename these flags to something more truthful like  `_previousRecordsExist` and `_nextRecordsExist`.

Also the description  would probably be easier to understand if it were something like "Flag for use in template to show/hide the previous page button." (to make it clear what previousPageLoader widget does).